secure_message/repository/retriever.py

```diff
@@ -340,8 +340,8 @@ def check_db_connection():
         try:
             SecureMessage().query.limit(1).all()
         except Exception as e:
-            logger.exception("No connection to database")
-            response = jsonify({"status": "unhealthy", "errors": str(e)})
+            logger.exception("No connection to database", errors=str(e))
```

Comment:
logging the error of a generic exception is not great

Reply:
It's not but it's an improvement on returning it through the API (CodeQL hated that). It's also an improvement on just logging "No connection to database" which I feel doesn't help us a ton.

It might be necessary to have a more comprehensive look at logging in this service if we've got stuff like this in there :)